src/Main.java

```diff
@@ -2,6 +2,75 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-                System.out.println("Hello and welcome!");
+
+        System.out.println("Задача 1:"); // Задача 1
+        var dog = 8.0;
+        var cat = 3.6;
+        var paper = 763789;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+        System.out.println(" ");
```

Comment:
в println можно ничего не писать
```suggestion
        System.out.println();
```